app/services/request_password_reset.rb

```diff
@@ -2,20 +2,14 @@
   def perform
     # To prevent revealing account existence, we don't treat a
```

Comment:
I feel like this entire comment is helpful but could also exist as a spec description, which would allow our specs to be doing the documentation... Example: https://github.com/thoughtbot/clearance/blob/master/spec/controllers/passwords_controller_spec.rb#L48